src/Microsoft.TemplateEngine.Edge/LocalizableStrings.Designer.cs

```diff
@@ -212,5 +350,50 @@ internal class LocalizableStrings {
                 return ResourceManager.GetString("SettingsLoader_Error_FailedToScan", resourceCulture);
             }
         }
+        
+        /// <summary>
+        ///   Looks up a localized string similar to Could not load template..
+        /// </summary>
+        internal static string TemplateCreator_TemplateCreationResult_Error_CouldNotLoadTemplate {
+            get {
+                return ResourceManager.GetString("TemplateCreator_TemplateCreationResult_Error_CouldNotLoadTemplate", resourceCulture);
+            }
+        }
+        
+        /// <summary>
+        ///   Looks up a localized string similar to Failed to create template, details: {0}..
+        /// </summary>
+        internal static string TemplateCreator_TemplateCreationResult_Error_CreationFailed {
+            get {
+                return ResourceManager.GetString("TemplateCreator_TemplateCreationResult_Error_CreationFailed", resourceCulture);
+            }
+        }
+        
+        /// <summary>
+        ///   Looks up a localized string similar to Destructive changes detected..
+        /// </summary>
+        internal static string TemplateCreator_TemplateCreationResult_Error_DestructiveChanges {
```

Comment:
Where is this used?

Reply:
fixed, used in TemplateCreation.InstantiateAsynx